Source/WebInspectorUI/UserInterface/Views/ResourceHeadersContentView.js

```diff
@@ -367,12 +367,12 @@ WI.ResourceHeadersContentView = class ResourceHeadersContentView extends WI.Cont
 
         let protocol = this._resource.protocol || "";
         let urlComponents = this._resource.urlComponents;
-        if (protocol.startsWith("http/1")) {
+        if (protocol === "http/1.1") {
```

Comment:
ditto having the oldest stuff be last

also, why did this change?

Reply:
As I described above, `http/1.0` has never been a valid value returned from `networkProtocolName`